tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -718,10 +599,7 @@ Select the date cateogory
 Generate table
     [Tags]  HappyPath
     user clicks element     id:filtersForm-submit
-
-Wait until the table is generated
-    [Tags]  HappyPath
-    user clicks button  Share your table
+    user waits until page contains    Share your table
```

Comment:
Extra space here

Reply:
Discussed this with Luke - throughout the UI tests we use more than two spaces to separate arguments and keywords. This can sometimes make it clearer that they're separate. I guess ideally we could keep this the same throughout the test, but it isn't that important so will just leave this.